eng/Tools.props

```diff
@@ -2,6 +2,9 @@
   <PropertyGroup>
     <MSBuildTreatWarningsAsErrors>false</MSBuildTreatWarningsAsErrors>
     <EnableAnalyzers>true</EnableAnalyzers>
+    <!-- Unset the repo tool manifest property in CI as we don't use repo tools there anyway,
+    until https://github.com/dotnet/sdk/issues/10938 is fixed. -->
+    <_RepoToolManifest />
```

Comment:
The comment says in `CI`, however, this operation is not conditioned to CI.

Reply:
Thanks, lol fixed.